### PR TITLE
WL-4080: Make expandability more obvious to users

### PR DESCRIPTION
**Pull request by nicholaswilson100**
for ox7

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
On the View Screen, this involves the whole of the h1 being clickable, having the mouse change to a pointer when hovering over it and having the colour change to a lighter blue when hovering over it.
On the Edit Screen, this means having the top bit of the h1 clickable and turning to a lighter blue and a mouse pointer when hovering over it.

**Comment by nicholaswilson100** [comment]
Also, for both screens ,we now have a white down/right arrow to indicate expanded / collapsed.

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 1]
```diff
@@ -515,7 +515,7 @@ private void displayCitations(PrintWriter out, List<Citation> citations, Citatio
 			} else {
 				// We only want to show the open url if no custom urls have been specified.
 				if (citation.getCitationProperty("otherIds") instanceof Vector) {
-					out.println("\t\t\t\t<a href=\"" + ((Vector) citation.getCitationProperty("otherIds")).get(0) + "\" target=\"_blank\">"
+					out.println("\t\t\t\t<a href=\"" + ((Vector) citation.getCitationProperty()getCitationProperty("otherIds")).get(0) + "\" target=\"_blank\">"
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: Has same issue we're just fixed.

**Comment by nicholaswilson100** [comment]
Can't be merged

**Comment by buckett** [comment]
Doesn't merge.

**Comment by nicholaswilson100** [comment]
Rebased

File: citations-tool/tool/src/webapp/css/citations.css [hunk 2]
```diff
@@ -173,10 +173,17 @@ ol.vertical li.placeholder::before {
 .descriptionButtons {
 	padding:5px;
 }
+ol.serialization li div.sectionEditor:hover {
+	cursor: pointer;
+	background: #2a4667 none repeat scroll 0 0;
```
Comment (buckett) [inline citations-tool/tool/src/webapp/css/citations.css:178]: Didn't we have colours coming from properties? Is this going to make it difficult to contribute back?

File: citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm [hunk 3]
```diff
@@ -33,19 +33,30 @@
...
+		#set( $linkClickId = 'linkClick' + $nestedSection.getLocation() )
+		#set( $toggleImgDiv = 'toggleImgDiv' + $nestedSection.getLocation() )
+		#set( $toggleImg = 'toggleImg' + $nestedSection.getLocation() )
 		#set( $toogleId = 'toggle' + $nestedSection.getLocation())
 		#set( $removeDivId = 'removeSection' + $nestedSection.getLocation() )
 		#set( $addSubsectionId = 'addSubsection' + $nestedSection.getLocation() )
 		#set( $addH1SubsectionButtonId = 'addH1SubsectionButton' + $nestedSection.getLocation() )
 		#set( $buttonsDiv = 'buttonsDiv' + $nestedSection.getLocation() )
 		#set( $num = $num + 1)
+		<li id='$linkId' class='months h1Editor' data-value='$nestedSection.getValue()' data-location='$nestedSection.getLocation()'
```
Comment (buckett) [inline citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm]: What's the class of `months` for?

File: citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm [hunk 4]
```diff
@@ -33,19 +33,30 @@
...
 			#set( $hasSectionsClass = ' hasSections')
 		#end
 		<li id='$linkId' class='months h1Editor $hasSectionsClass' data-value='$nestedSection.getValue()' data-location='$nestedSection.getLocation()'
 			data-sectiontype='$nestedSection.getSectiontype()'>
-			<div id='$editorDivId' class='editor h1Editor sectionEditor'>
-				$!nestedSection.getValue()
+			<div id="$linkClickId" class="h1TopBar">
+				#if ($nestedSection.getChildren().size() > 0)
+					<div id="$toggleImgDiv"><img border="0" width="16" height="16" align="top" id="$toggleImg"
+							class="toggleIcon accordionArrow" style="padding-top: 11px;" src="/library/image/sakai/white-arrow-right.gif" alt="Citation View">
```
Comment (buckett) [inline citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm]: i18n on Citations View

File: citations-tool/tool/src/webapp/css/citations.css (hunk 2 above)
Comment (nicholaswilson100) [inline citations-tool/tool/src/webapp/css/citations.css:178]: There isn't any inline-style equivalent for :hover I don't think.  I could do something clever in the js...

Comment (nicholaswilson100) [inline citations-tool/tool/src/webapp/css/citations.css:178]: ...or just remove it when contributing back.

**Pull request merged**